common/src/main/java/io/quarkus/ts/openshift/common/OpenShiftTestExtension.java

```diff
@@ -1,30 +1,7 @@
 package io.quarkus.ts.openshift.common;
 
-import io.fabric8.knative.client.KnativeClient;
-import io.fabric8.openshift.api.model.ImageStream;
-import io.fabric8.openshift.api.model.Route;
-import io.fabric8.openshift.client.OpenShiftClient;
-import io.quarkus.ts.openshift.app.metadata.AppMetadata;
-import io.quarkus.ts.openshift.common.actions.OnOpenShiftFailureAction;
-import io.quarkus.ts.openshift.common.config.Config;
-import io.quarkus.ts.openshift.common.injection.InjectionPoint;
-import io.quarkus.ts.openshift.common.injection.TestResource;
-import io.quarkus.ts.openshift.common.injection.WithName;
-import io.quarkus.ts.openshift.common.util.AwaitUtil;
-import io.quarkus.ts.openshift.common.util.OpenShiftUtil;
-import io.restassured.RestAssured;
-import org.junit.jupiter.api.extension.AfterAllCallback;
-import org.junit.jupiter.api.extension.BeforeAllCallback;
-import org.junit.jupiter.api.extension.BeforeEachCallback;
-import org.junit.jupiter.api.extension.ExtensionContext;
-import org.junit.jupiter.api.extension.ExtensionContext.Namespace;
-import org.junit.jupiter.api.extension.ExtensionContext.Store;
-import org.junit.jupiter.api.extension.LifecycleMethodExecutionExceptionHandler;
-import org.junit.jupiter.api.extension.ParameterContext;
-import org.junit.jupiter.api.extension.ParameterResolutionException;
-import org.junit.jupiter.api.extension.ParameterResolver;
-import org.junit.jupiter.api.extension.TestExecutionExceptionHandler;
-import org.junit.jupiter.api.extension.TestInstancePostProcessor;
+import static org.fusesource.jansi.Ansi.ansi;
+import static org.junit.platform.commons.util.AnnotationUtils.findAnnotatedFields;
```

Comment:
Keep the dependency order :)

Reply:
I'm using the quarkus formatter and it's being a pain when I modify the source code. Will keep the dependency order.

Reply:
Hey guys, original author of current import sorting configuration. I feel very strongly about the Quarkus formatter (not in a good way), but I also no longer contribute to this test suite, so feel free to move towards the Quarkus formatter if you wish :-)

Reply:
I've updated my IDE to not automatically sort the imports, so this issue won't happen to me anymore. 
About the Quarkus formatter, up to the rest of the team

Reply:
Already fixed.